spec/fixtures/generators-list.js

```diff
@@ -0,0 +1,75 @@
+export default function generatorsList() {
```

Comment:
NABD, but you could expose the array here instead of a function like `export default []`.